### ISSUE TITLE
NVDA doesn't read the slider that indicates the UAC level

### ISSUE THREAD

**Issue by mdlawler** [nvaccess/nvda#18916]
### Steps to reproduce:
In the start menu search box type "change user account" to get change user account control settings and press enter.  Focus the slider to change the level of UAC.  Use up and down arrow and the slider won't be read.  Using nvda-tab does read the current value.  Narrator is also able to read the slider with up and down arrow as it changes.
### Actual behavior:

The slider value isn't read with up and down arrow.
### Expected behavior:

The slider should be read with up and down arrow.
#### NVDA installed/portable/running from source:
NVDA is installed.
#### NVDA version:
alpha-52562,3d74061b (2026.1.0.52562)
#### Windows version:
Windows 11 [Version 10.0.26100.6584]
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I've seen this with many alpha versions and have been told that it happens in 2025.2.
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by tareh7z** [comment]
Hi!
This is a duplicate of [#9356](https://github.com/nvaccess/nvda/issues/9356) and [#9669](https://github.com/nvaccess/nvda/issues/9669).

**Comment by mdlawler** [comment]
And so it is thanks.  I'm not the best at searching issues.
